tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/DockerTestFramework.cs

```diff
@@ -0,0 +1,337 @@
+// <copyright file="DockerTestFramework.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers.AutoInstrumentation.Containers;
+using Datadog.Trace.TestHelpers.FluentAssertionsExtensions;
+using DotNet.Testcontainers.Configurations;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Datadog.Trace.TestHelpers.AutoInstrumentation
+{
+    public class DockerTestFramework : XunitTestFramework
```

Comment:
Surely we can reuse some of our existing FX rather than duplicating everything? I hope? 😬

Reply:
Oh good call. I just duplicated everything to make sure it worked, and forgot to refactor